Mod2/Lessons/Week5/ManyToManyWithAnORM.md

```diff
@@ -0,0 +1,64 @@
+# Many-to-Many With an ORM
+
+## Learning Goals
+*  Understand why frameworks have versions
+*  Reinforce how to structure many-to-many database relationships
+*  Implement a many-to-many relationship using Entity Framework
+
+We're going to continue building our plant tracker application, let's all go ahead and open that app.
+
+### A Bit About Versions
+
+Our goal for today is to build a many-to-many relationship using Entity Framework, and it's a good time to be alive because a recent version of Entity Framework Core introduced a simpler way to do this than was available in the past!
+
+Let's take this opportunity to learn more about software versioning.
+
+❓Where have you seen the term `version` with regards to software development? What is your current understanding of what a software version is?
+
+<details><summary>One way you might answer these questions..</summary>
+When we start up a new project we pick what version we want to use. Also, when we install a package we see the versions for everything installed.
+
+A software version is a grouping of updates. Having version numbers allows everyone to see what was updated and when.
+</details>
+<br>
+
+❓Why might the teams building frameworks use versions instead of just releasing new features as they build them?
+
+<details><summary>Some of the many reasons..</summary>
+
+- Sometimes an update creates what's called a `breaking change` where with the new version some old code won't work anymore. You don't want your software to break without warning!
+- It's easier for users to see what's new by viewing `release notes`.
+- Developers can release `beta versions` for early adopters to try out the latest features and find bugs. Then they can later make that version a `stable version` that is a well-tested version most users will prefer.
+</details>
+
+<br>
+
+When the development team releases a new version of their software, they also create release notes. Let's take a look at the release notes for EF Core 5.0: https://learn.microsoft.com/en-us/ef/core/what-is-new/ef-core-5.0/whatsnew
+
+
+Something to be aware of when you're looking up tutorials or blog posts is that sometimes you will find tutorials that have not been updated and are still referencing an old version. For example, this tutorial is still teaching the old way of building a many-to-many relationship in EF. https://www.entityframeworktutorial.net/efcore/configure-many-to-many-relationship-in-ef-core.aspx
+
```

Comment:
I might make a note here that if you are researching and an answer is not designated as part of a particular version, proceed with caution!